portal/src/graphql/portal/SingleSignOnConfigurationScreen.tsx

```diff
@@ -258,7 +273,7 @@ const SingleSignOnConfigurationScreen1: React.VFC<{
     };
   }, [renderToString]);
 
-  if (form.isLoading || featureConfig.loading) {
+  if (form.isLoading || featureConfig.loading || isEffectiveSecretLoading) {
```

Comment:
We should handle this like `if (form.isLoading || featureConfig.loading || effectiveSecretConfig.loading)` and handle the error too `if (form.loadError ?? featureConfig.error ?? effectiveSecretConfig.error)`